### PR TITLE
Add more tests

### LINKED ISSUE #119: Add more tests
### Motivation

Our test coverage is currently surprisingly good - but it would be nice to at least add some parametrized tests for the ward managemen and timeline views, as well as for assignment operations

### PR DESCRIPTION
**Pull request by charludo**
### Short description

This PR brings test coverage up to 80% 🎉 

### Proposed changes

- add tests to untested views
- tweak some user permissions

### Side effects

- none I am aware of

@noworneverev do we still need the `/bed-assignments/` view? As far as I can tell, it is currently broken anyways.
Do you think the patient detail view introduced in #103 is a sufficient replacement?

### Resolved issues

Fixes: #

### CODE REVIEW

File: ycms/cms/views/authentication/registration_view.py
Comment (noworneverev) [inline ycms/cms/views/authentication/registration_view.py:1]: I tried to create a new user, but an error occurred, stating that `null value in column "ward_as_timeline" of relation "cms_user" violates not-null constraint`. Will this error be resolved after merging the timeline PR?

**Comment by noworneverev** [comment]
> ### Short description
> This PR brings test coverage up to 80% 🎉
> 
> ### Proposed changes
> * add tests to untested views
> * tweak some user permissions
> 
> ### Side effects
> * none I am aware of
> 
> @noworneverev do we still need the `/bed-assignments/` view? As far as I can tell, it is currently broken anyways. Do you think the patient detail view introduced in #103 is a sufficient replacement?
> 
> ### Resolved issues
> Fixes: #

I think there's no need the `/bed-assignments/` view. Do you want to just remove them in this PR, or I create another one to remove them?

File: ycms/cms/views/authentication/registration_view.py
Comment (charludo) [reply ycms/cms/views/authentication/registration_view.py:1]: Could you try pruning the db? That field was introduced in the Timeline PR, which this isn't rebased onto yet.

**Comment by charludo** [comment]
> I think there's no need the `/bed-assignments/` view. Do you want to just remove them in this PR, or I create another one to remove them?

Your choice!

File: ycms/cms/views/authentication/registration_view.py
Comment (noworneverev) [reply ycms/cms/views/authentication/registration_view.py:1]: Ok, after I rebuild the db, the error is fixed now.

**Comment by noworneverev** [comment]
> > I think there's no need the `/bed-assignments/` view. Do you want to just remove them in this PR, or I create another one to remove them?
> 
> Your choice!

I'll create another PR then. I think this PR is good to go after solving the conflict. 😄

**Review by noworneverev — APPROVED** [review]
🎉

**Pull request merged**